playground/nuxt.config.ts

```diff
@@ -44,6 +44,11 @@ export default <NuxtConfig> {
     imgix: {
       baseURL: 'https://assets.imgix.net'
     },
+    imgproxy: {
+      baseURL: 'https://cc.cz/',
+      key: 'xxxxxxxxxxxxxx',
+      salt: 'xxxxxxxxxxxxxx'
```

Comment:
Provider options are also included in the pre-built JS that the web browser downloads to generate the image URL on the client side. Therefore, it is possible for a third party to steal the key and salt, but is it safe to leak these values?

Reply:
> Provider options are also included in the pre-built JS that the web browser downloads to generate the image URL on the client side. Therefore, it is possible for a third party to steal the key and salt, but is it safe to leak these values?

@misaon This PR is implementing https://github.com/nuxt/image/issues/378#issuecomment-898370646

Do you have any insight for the above?

Reply:
Related to https://github.com/nuxt/image/issues/276